src/commands/fun/latex.ts

```
@@ -0,0 +1,82 @@
+import { ApplicationCommandOptionData, CommandInteraction, MessageEmbed, MessageAttachment } from 'discord.js';
+import fetch from 'node-fetch';
+import { createCanvas, loadImage } from 'canvas';
+import { Command } from '@lib/types/Command';
+
+const BACKGROUND_COLOR = `rgb(${255 - 47}, ${255 - 49}, ${255 - 54})`;
+const PADDING = 4;
+
+export default class extends Command {
+
+	// Made by Brendan Lewis (@craftablescience)
+
+	description = `Accepts a LaTeX expression and posts it as a rendered image.`;
+
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'input',
+			description: 'The LaTeX expression to render',
+			type: 'STRING',
+			required: true
+		}
+	]
+
+	async tempRun(interaction: CommandInteraction): Promise<void> {
+		const tex = interaction.options.getString('input');
+		const requestURL = `https://latex.codecogs.com/svg.json?${encodeURIComponent(tex)}`;
+		const errorResponse = "Sorry, I couldn't render that LaTeX expression.";
+		let imageAsBase64: Buffer;
+		try {
+			const response = await fetch(requestURL, { method: 'Get' });
+			const imageAsBase64JSON = await response.json();
+			imageAsBase64 = Buffer.from(imageAsBase64JSON.latex.base64, 'base64');
+		} catch (error) {
+			return interaction.reply({ content: errorResponse, ephemeral: true });
```

You might want to throw the error here too so it can be logged.